ui/app/package.json

```diff
@@ -51,7 +51,11 @@
     "parserOptions": {
       "ecmaVersion": 2020
     },
-    "rules": {},
+    "rules": {
+      "@typescript-eslint/no-unused-vars": "off",
+      "@typescript-eslint/no-explicit-any": "off",
+      "@typescript-eslint/explicit-module-boundary-types": "off"
```

Comment:
Why these changes?

Reply:
Because they were delivering weird warnings in vetur about types not being used even though they were being used. This has to do with veturs poor TS support.